main/User_config.h

```diff
@@ -287,4 +287,7 @@ uint8_t wifiProtocol = 0; // default mode, automatic selection
 /*-------------------DEFINE LOG LEVEL----------------------*/
 #define LOG_LEVEL LOG_LEVEL_NOTICE
 
+#define convertTemp_CtoF(c) ((c * 1.8) + 32)
+#define convertTemp_FtoC(f) ((f - 32) * 5/9)
```

Comment:
why are these macros part of the user_config.h and not in the main.ino, they are not configurable? Also what would be the different when using a function instead of a macro?

Reply:
I was thinking that `user_config.h` would be more accessible to all of the sensors, but you're right; it should be in `main.ino` instead.

The main benefit in using a macro instead of a function is that the macro doesn't need have the variable type defined. Meaning it doesn't matter if an int, double, or float called with the macro. (Otherwise, there would need to be a function defined for each type.)
From my understanding, this is because the compiler basically copies the code block to be inline with where it's being called from.
BT devices are using 'double' and AHT is using 'float' values.

Reply:
I moved it to `main.ino` and pushed the change.